doc/content/guides/getting-started/cli/_index.md

```diff
@@ -40,6 +40,22 @@ The command-line needs to be configured to connect to your deployment on `thethi
 
 > NOTE: When using the snap packages, `~/.ttn-lw-cli.yml` will fail with permission errors. Choose a different path.
 
+### Generate configuration file
+
+For a standard deployment on `thethings.example.com`, all you need is:
+
+```bash
+$ ttn-lw-cli use thethings.example.com [--fetch-ca] [--user] [--overwrite]
+```
+
+This will generate and save the required CLI config file. Use the `--fetch-ca` flag to also connect to the server and retrieve the CA required for establishing secure communication. By default, the file is saved on the current directory, use the `--user` to save it under the user config directory.
```

Comment:
Note that `--fetch-ca` is only necessary when using a CA that isn't already trusted by the system.